apps/proxy/src/styles.scss

```diff
@@ -62,7 +62,6 @@
 .app-content {
     padding: 16px;
     min-height: 100%;
-    background-color: var(--color-common-bg);
```

Comment:
why done this

Reply:
this background color overwrite by line 68 background color